### PR TITLE
Add link command

### PR DESCRIPTION
**Pull request by karthickkc**
Links a Person to a Concert

### CODE REVIEW

**Review by wes-w-z-h — CHANGES_REQUESTED** [review]
Insufficient tests, need a new issue for storage of concert contacts

File: src/test/java/seedu/address/logic/commands/LinkCommandTest.java
Comment (wes-w-z-h) [inline src/test/java/seedu/address/logic/commands/LinkCommandTest.java:1]: why is it all commented out?

File: src/test/java/seedu/address/logic/parser/ListCommandParserTest.java
Comment (wes-w-z-h) [inline src/test/java/seedu/address/logic/parser/ListCommandParserTest.java:1]: list command parser or link command parser

File: src/test/java/seedu/address/testutil/TypicalIndexes.java [hunk 1]
```diff
@@ -9,4 +9,6 @@ public class TypicalIndexes {
     public static final Index INDEX_FIRST_PERSON = Index.fromOneBased(1);
     public static final Index INDEX_SECOND_PERSON = Index.fromOneBased(2);
     public static final Index INDEX_THIRD_PERSON = Index.fromOneBased(3);
+
```
Comment (wes-w-z-h) [inline src/test/java/seedu/address/testutil/TypicalIndexes.java]: dont need a new line here

**Review by wes-w-z-h — CHANGES_REQUESTED** [review]

**Comment by wes-w-z-h** [comment]
image
Uncaught exception for invalid index

**Review by wes-w-z-h — CHANGES_REQUESTED** [review]
image
Doesn't do massOps so this command should throw an error, can consider adding more test cases to catch this sort of inputs including invalid c/INDEX

File: src/main/java/seedu/address/logic/commands/LinkCommand.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+ */
+public class LinkCommand extends Command {
+
+    public static final String COMMAND_WORD = "link";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": links the person identified "
+            + "by the index number used in the displayed person list to the concert identified by the"
+            + "index number used in the displayed concert list.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+             + PREFIX_CONCERT + "INDEX (must be a positive integer)\n"
```
Comment (hj235) [inline src/main/java/seedu/address/logic/commands/LinkCommand.java]: Can u change the text to PERSONINDEX and CONCERTINDEX

**Review by wes-w-z-h — APPROVED** [review]
lgtm 🥳🤩

**Pull request merged**